.travis.yml

```diff
@@ -2,7 +2,17 @@ language: generic
 
 script: bash ./deploy.sh
 
+branches:
+  only:
+    - gh-pages
+
 env:
   global:
   - ENCRYPTION_LABEL: 104fbe69e8fa
   - COMMIT_AUTHOR_EMAIL: travis-ci@w3.org
+  - URL="https://w3c.github.io/permissions/"
+  - DECISION="https://lists.w3.org/Archives/Public/public-webappsec/2015Mar/0170.html"
+  - secure: SneznWMBm52ct+mFI6NAVrB8k/BjO/H/hjR1eB3tjPwF0vzpL5fnbD1mmlNmVJMTWMsSUfI917JotA+7lfwoWJuHwlq2WfzQ0ybfvtuqjo7LS22T5X8IFYmCyVNR5asbJha/LNhqUyGDsZlPBgbKK0hqTd8eWjjQTztnlSrorGM=
+
+after_success:
+  - '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && curl "https://labs.w3.org/echidna/api/request" --data "url=$URL" --data "decision=$DECISION" --data "token=$TOKEN"'
```

Comment:
> I've been defending some of the content here by saying that we'll need to ask for consensus on it later, but if this is going to start auto-updating https://www.w3.org/TR/permissions/ and calling it a Working Draft, I think we're going to get some objections.

Yeah, that's why I thought that only if something hits "gh-pages" should it call echidna. However, I think it's better to only have one version of the spec (even if the stuff is controversial) - controversial stuff should just be marked as such with an issue and a call for people to collaborate on reaching consensus... or else we risk never being able to get the spec on TR because there is likely always going to be controversy.